docs/source/38_0.rst

```diff
@@ -0,0 +1,135 @@
+===============
+38.0: Love, Ken
+===============
+
+Sprint Theme
```

Comment:
On Wed, Jul 20, 2016 at 04:50:22AM -0700, Amador Pahim wrote:
> > @@ -0,0 +1,135 @@
> > +===============
> > +38.0: Love, Ken
> > +===============
> > +
> > +Sprint Theme
> 
> @adereis, how about this?
> https://github.com/apahim/avocado/blob/release_notes_v2/docs/source/38_0.rst

LGTM

> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/avocado-framework/avocado/pull/1314/files/b43e323a2ef2fb1620cda3e2011147fe53683219#r71510613

-- 
Ademar Reis
Red Hat

^[:wq!